app/controllers/structuresbuildingallowance/SbaClaimsController.scala

```diff
@@ -65,15 +69,37 @@ class SbaClaimsController @Inject()(
       form.bindFromRequest().fold(
         formWithErrors =>
           Future.successful(BadRequest(view(formWithErrors, list, taxYear, request.user.isAgentMessageKey))),
-
         value =>
           for {
             updatedAnswers <- Future.fromTry(request.userAnswers.set(SbaClaimsPage, value))
             _ <- sessionRepository.set(updatedAnswers)
+            _ <- auditSBAClaims(taxYear, request)
```

Comment:
audit should be sent or not according to the value, weather the user wants to add new sba or not.

Reply:
Yes I see what you mean - you are right @modivilioglu